docker-compose.yml

```
@@ -3,12 +3,12 @@ version: "3"
 services:
   dev:
     image: "todoapp:dev"
-    build: .
+    build: 
+      context: .
+      target: development
     ports:
       - "5000:5000"
     env_file: 
       - ".env"
     volumes:
-     - type=bind
-       source=/todo_app
-       target=/app/todo_app
\ No newline at end of file
+     - ./todo_app:/todo_app
```

I suspect you made this change before tweaking the Dockerfile - it doesn't quite line up with your latest!
```suggestion
     - ./todo_app:/app/todo_app
```